underlay/src/main/resources/config/display/sd/criteriaselector/icd10CM/startDateGroupByCount.json

```diff
@@ -1,4 +1,29 @@
 {
   "groupByCount": true,
-  "attribute": "start_date"
-}
\ No newline at end of file
+  "attributes": {
+    "conditionOccurrence": {
+      "values": [
+        "start_date",
+        "condition"
```

Comment:
The JOIN column/attribute for the ICD codes is `source_criteria_id` (which maps to e.g. `condition_source_concept_id`), instead of the domain-named attribute e.g. `condition` (which maps to e.g. `condition_concept_id`) that the standard codes use.

For CPT-4 and PheWAS, you can use the same `source_criteria_id` attribute here. Thinking about this now though, we probably need to do some additional testing for those two because I think it's possible to have multiple `source_concept_id` values per code. I don't know if that actually happens, but the way the config SQL is written it is possible. If so, we'd need to do something further to address that -- maybe adding an attribute for the CPT-4 and PheWAS codes to the occurrence entities and then specifying that here.

Reply:
Changed all of the source entities to `source_criteria_id`. CPT-4 and PheWAS issues to be investigated and fixed separately per offline discussion.